cache/solidity-files-cache.json

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
Looks like this file was accidentally committed—we should add this cache folder to the gitignore